documentation/api/dump.md

```diff
@@ -0,0 +1,92 @@
+# Dump - Get
+
+Captures a managed dump of a specified process without using a debugger.
+
+## HTTP Route
+
+```http
+GET https://localhost:52323/dump/{pid}?type={type}
+```
+
+or 
+
+```http
+GET https://localhost:52323/dump/{uid}?type={type}
+```
+
+or
+
+```http
+GET https://localhost:52323/dump?type={type}
+```
+
+> **NOTE:** Process information (IDs, names, environment, etc) may change between invocations of these APIs. Processes may start or stop between API invocations, causing this information to change.
```

Comment:
Do we already have the change merged where we only allow one concurrent dump collection API call? Might be worth calling out the limitation

Reply:
We should also mention the observability overhead of collecting a dump which results in the entire process being suspended